src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Tools.Add.PackageReference
+{
+    internal class LocalizableStrings
```

Comment:
@livarcocc @srivatsn : 
What about the remaining commands i.e. `dotnet remove|list|update package` ?  They would also contain strings that need localization.